lib/Doctrine/Record/Filter/Compound.php

```diff
@@ -32,38 +32,48 @@
  */
 class Doctrine_Record_Filter_Compound extends Doctrine_Record_Filter
 {
+    /**
+     * @var string[]
+     */
     protected $_aliases = array();
 
+    /**
+     * @param string[] $aliases A list of relation name
+     */
     public function __construct(array $aliases)
     {
         $this->_aliases = $aliases;
     }
 
+    /**
+     * @throws Doctrine_Table_Exception
+     */
     public function init()
     {
-    	// check that all aliases exist
-    	foreach ($this->_aliases as $alias) {
+        // check that all aliases exist
+        foreach ($this->_aliases as $alias) {
             $this->_table->getRelation($alias);
-    	}
+        }
     }
 
     /**
-     * filterSet
-     * defines an implementation for filtering the set() method of Doctrine_Record
+     * Provides fallback for setting value.
+     *
+     * @return Doctrine_Record the given record
      *
-     * @param mixed $name                       name of the property or related component
```

Comment:
We should keep the phpdoc here, and define it as `string` (as aliases are string only)